src/super_gradients/common/auto_logging/auto_logger.py

```diff
@@ -32,7 +32,7 @@ def _setup_default_logging(self, log_level: str = None) -> None:
         # Therefore the log file will have the parent PID to being able to discriminate the logs corresponding to a single run.
         timestamp = time.strftime("%Y_%m_%d_%H_%M_%S", time.localtime())
         self._setup_logging(
-            filename=os.path.expanduser(f"~/sg_logs/logs_{os.getppid()}_{timestamp}.log"),
+            filename=os.path.join(os.environ.get("SUPER_GRADIENTS_LOG_DIR", default="~/sg_logs"), f"logs_{os.getppid()}_{timestamp}.log"),
```

Comment:
It would be great if you can add it to the env variables [here](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/common/environment/env_variables.py)

```python
    @property
    def SUPER_GRADIENTS_LOG_DIR(self) -> str:
        return os.getenv("SUPER_GRADIENTS_LOG_DIR", default="~/sg_logs")
```

And then directly use it
```python
filename =os.path.join(env_variables.SUPER_GRADIENTS_LOG_DIR, f"logs_{os.getppid()}_{timestamp}.log")
```

Reply:
Thanks for the feedback, I am using the env var in both instances through env_variables.py now